frontend/test/metabase/scenarios/cross-version/source/02-datamodel.cy.spec.js

```diff
@@ -0,0 +1,113 @@
+import { SAMPLE_DB_ID } from "__support__/e2e/cypress_data";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { ORDERS, ORDERS_ID, REVIEWS, PRODUCTS, PEOPLE } = SAMPLE_DATABASE;
+
+const sampleDBDataModelPage = `/admin/datamodel/database/${SAMPLE_DB_ID}`;
+
+it("should configure data model settings", () => {
+  cy.signInAsAdmin();
+
+  cy.visit("/admin/datamodel");
+  cy.location("pathname").should("eq", sampleDBDataModelPage);
```

Comment:
Is this completely safe without any `cy.findByText` before hand (to ensure a successful navigation)?